### PR TITLE
Onboarding updates for Product Hunt

### PR DESCRIPTION
**Pull request by swoodward90**
## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: pages/getting-started/admin-console.md [hunk 1]
```diff
@@ -5,14 +5,14 @@ description: Supercharge your day two operations.
...
-The Plural admin console is a web application that serves as a control panel for all your Plural applications. It:
+The Plural Admin Console is a web application that serves as a control panel for all your Plural applications. It:
 
 * manages automated upgrades delivered from the Kubernetes api
 * serves as a thin Grafana to visualize application metrics and logs
 * serves as a built-in K8s dashboard for all plural apps in the cluster, along with providing app-level health checking
 * is the touchpoint at which incidents can be filed with the owner of an application
 
-The console is not a strict dependency, but we highly recommended installing it. It can be installed and deployed like any other application on Plural.
+We highly recommended installing Plural Console. It can be installed and deployed like any other application on Plural.
```
Comment (nickcernera) [inline pages/getting-started/admin-console.md:15]: We might want to change the wording here to reflect the fact that Plural Console is currently required in our Cloud Shell onboarding.

File: pages/getting-started/cloud-shell-quickstart.md [hunk 2]
```diff
@@ -18,118 +18,87 @@ in under 30 minutes, all within your browser.
...
 
 ### Set up a Workspace
 
 We'll now start setting up your Kubernetes cluster configuration. Here's a guide to the config:
 
-**cluster**: The name of your Kubernetes cluster. Name this based on what you're planning to run.
+**cluster**: The name of your Kubernetes cluster. Name this based on what you're planning to run. Feel free to use the provided default.
 
-**bucket prefix**: We provision S3 buckets for storing logs and state. Just enter any string to help us uniquely name your buckets.
+**bucket prefix**: We provision S3 buckets for storing logs and state. Enter any string to help us uniquely name your buckets, or use the provided default.
```
Comment (nickcernera) [inline pages/getting-started/cloud-shell-quickstart.md:39]: I don't think we provide a default beyond the placeholder? Could be wrong here..

**Review by nickcernera — CHANGES_REQUESTED** [review]
Left a few minor comments that need adjustments, but good to go otherwise. Looks great!

File: pages/getting-started/admin-console.md (hunk 1 above)
Comment (swoodward90) [reply pages/getting-started/admin-console.md:15]: Yeah it's not required in CLI onboarding but can update

File: pages/getting-started/cloud-shell-quickstart.md (hunk 2 above)
Comment (swoodward90) [reply pages/getting-started/cloud-shell-quickstart.md:39]: Yeah I was going to ask about that - why don't we persist the placeholder? Are we just showing that it shouldn't just be plural, but not actually giving them the option to select that?

File: pages/getting-started/admin-console.md (hunk 1 above)
Comment (swoodward90) [reply pages/getting-started/admin-console.md]: Updated

File: pages/getting-started/cloud-shell-quickstart.md (hunk 2 above)
Comment (swoodward90) [reply pages/getting-started/cloud-shell-quickstart.md]: Removed

**Review by nickcernera — APPROVED** [review]
LGTM

**Pull request merged**